evalai/challenges.py

```diff
@@ -2,8 +2,52 @@
 
 from click import echo
 
+from evalai.utils.challenges import (
+                                    get_challenge_list,
+                                    get_past_challenge_list,
+                                    get_future_challenge_list,)
 
-@click.command()
-def challenges():
-    """Example script."""
-    echo('Hello Challenges!')
+
+@click.group(invoke_without_command=True)
+@click.pass_context
+def challenges(ctx):
+    """
+    Challenges and related Options.
+    """
+    if ctx.invoked_subcommand is None:
+        welcome_text = """Welcome to the EvalAI CLI. Use evalai challenges --help for viewing all the options"""
+        echo(welcome_text)
+
+
+@click.group(invoke_without_command=True, name='list')
+@click.pass_context
+def list_challenges(ctx):
+    """
+    Lists all challenges.
+    """
+    if ctx.invoked_subcommand is None:
+        get_challenge_list()
+
+
+@click.command(name='past')
+def list_past_challenges():
+    """
+    Lists past challenges.
+    """
+    get_past_challenge_list()
+
+
+@click.command(name='future')
+def list_future_challenges():
+    """
+    Lists future challenges.
+    """
+    get_future_challenge_list()
+
```

Comment:
please add a command to view the ongoing challenges.

Reply:
@RishabhJain2018 Isn't ongoing challenge the same as listing all active challenges?